### PR TITLE
Improve display of Labels/Projects/Assignees sort options

### PR DESCRIPTION
**Pull request by yp05327**
Labels:
Before: (no highlights)
image
After:
image
image

Projects:
Before: (no highlights)
image

After:
image
image

Assignee:
Before: (no highlights)
image
After:
image
image

### CODE REVIEW

**Review by lunny — APPROVED** [review]

**Review by puni9869 — APPROVED** [review]

File: routers/web/repo/issue.go [hunk 1]
```diff
@@ -174,7 +174,11 @@ func issues(ctx *context.Context, milestoneID, projectID int64, isPullOption uti
 	// 0 means issues with no label
 	// blank means labels will not be filtered for issues
 	selectLabels := ctx.FormString("labels")
-	if len(selectLabels) > 0 {
+	if selectLabels == "" {
+		ctx.Data["AllLabels"] = true
+	} else if selectLabels == "0" {
+		ctx.Data["NoLabel"] = true
```
Comment (delvh) [inline routers/web/repo/issue.go:180]: That sounds like a bad idea. `0` sounds like a label someone could use.
Can't we use something that no one ever uses, i.e. in the private use area?

Comment (delvh) [reply routers/web/repo/issue.go:180]: Or an empty string or something like that?

Comment (yp05327) [reply routers/web/repo/issue.go:180]: But it is label's id, not label's name.
Or we can move this `if` condition to L187? After generated `labelIDs` in the below.

**Comment by puni9869** [comment]
Let's ship this PR.
🚀

**Pull request merged**